Library/Homebrew/formula.rb

```diff
@@ -1014,6 +1016,13 @@ def service_name
   # The generated launchd {.plist} file path.
   sig { returns(Pathname) }
   def plist_path
+    odeprecated "formula.plist_path", "formula.launchd_service_path"
```

Comment:
Can't add until next minor version. comment it out and add a TODO above and I'll add it then.